Dockerfile

```diff
@@ -0,0 +1,30 @@
+#  Alfresco Setup build
+
+FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/ark_base:latest
+
+LABEL ORG="Armedia LLC" \
+      APP="Alfresco Setup" \
+      VERSION="1.0" \
+      IMAGE_SOURCE=https://github.com/ArkCase/ark_alfresco_setup \
+      MAINTAINER="Armedia LLC"
+
+WORKDIR /app
+
+# Set default docker_context.
+ARG resource_path=artifacts
+
+COPY ${resource_path}/setup_alfresco.py .
+COPY ${resource_path}/alfresco_setup.properties .
```

Comment:
There is a problem here: The Alfresco configuration to apply is embedded inside the container. That means that each time you want to have a different configuration (even a very small change), you will need to build a new container. I think you should design this alfresco_setup container such that it expects the properties file to be mounted inside the container. 

When running in Kubernetes/Helm, this will most probably take the form of a ConfigMap. So the Alfresco setup will truly be a input to the Helm chart and can easily be tuned for each deployment.

Reply:
Good point.  We decided to convert it into 9 environment variables